client/src/components/NavMenu/NavMenu.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+import './NavMenu.css'
+
+const NavMenu = (props) => (
+    props.isVisible ?
```

Comment:
This is a very clever and effective way of doing this. However, having the entire component's JSX inside the ternary makes it difficult to read. Could you move the ternary to line 7, where you have the `nav-container` class? Typically, we use a CSS class as a flag, so `.nav-container` would be the default state (invisible) and `.nav-container.is-visible` would be the visible state. Does that make sense?

Reply:
I got it. I just made the change. I had to comment out the storybook router as it keeps throwing errors since it hasn't been added to config yet.